flink-container/src/test/java/org/apache/flink/container/entrypoint/StandaloneJobClusterEntryPointTest.java

```diff
@@ -67,54 +65,109 @@ public void testDontOverwriteExecutionMode() {
 	}
 
 	@Test
-	public void configuredJobIDTakesPrecedenceWithHA() {
-		Optional<JobID> jobID = Optional.of(JobID.generate());
+	public void configuredJobIdTakesPrecedenceWithHA() {
```

Comment:
Not required here, but imo, those tests that check precedence order would be better if they are rewritten as one test that checks all cases as table input.